### ISSUE TITLE
have 3 MDSs (1 up while 2 in standby) for the backend ceph cluster

### ISSUE THREAD

**Issue by dparmar18** [samba-in-kubernetes/sit-environment#25]
I'm not sure if I might be missing out on some code that I didn't look at but I was going through https://github.com/samba-in-kubernetes/sit-environment/pull/17 and it seems there's just one MDS being created, right? I'd say have at least 3 which makes it easy to handle any workload and/or any failure/crash in active MDS.

**Comment by obnoxxx** [comment]
Thanks for this suggestion, @dparmar18, this is very useful!

we can certainly look into adding this enhancement.
@xhernandez , @phlogistonjohn  - what do you think?

**Comment by xhernandez** [comment]
What's exactly the purpose of the additional MDS ?

If they are there just in case of a failure of the active MDS, I would prefer to catch that and see the failure. The purpose of this test environment is to identify functional issues in both Samba and CephFS to fix them and improve both projects, not to create a highly resilient environment (specially if this could hide some unwanted failures).

**Comment by spuiuk** [comment]
I agree with Xavi here. We are not testing cephfs directly but the cephfs over SMB and for that we don't really require 3 MDS machines which are required for high availability. That will simply take more resources and is not really tested.
